src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommandParser.cs

```diff
@@ -13,14 +13,23 @@ internal static class WorkloadListCommandParser
 
         public static readonly Option VerbosityOption = CommonOptions.VerbosityOption();
 
-        public static readonly Option VersionOption = WorkloadUpdateCommandParser.VersionOption;
+        public static readonly Option TargetSdkVersionOption = new Option<string>("--target-sdk-version", Workloads.Workload.Install.LocalizableStrings.VersionOptionDescription)
+        {
```

Comment:
Other commands use `--sdk-version`, can we use that here too to be consistent?

Reply:
`--sdk-version` is too vague, people would think it is the current sdk version.

Reply:
Can we normalize all the other commands to use that name then? And then we just have to declare it once and then reuse it, like the other options?

Reply:
`--sdk-version` is the right thing to use